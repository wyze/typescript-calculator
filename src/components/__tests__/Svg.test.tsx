import { h } from 'preact'
import Svg from '../Svg'
import snapshot from './utils/snapshot'

describe('<Svg />', () => {
  it('renders', () => snapshot(<Svg />))
  it('renders with children', () => snapshot(<Svg><g /></Svg>))
})
