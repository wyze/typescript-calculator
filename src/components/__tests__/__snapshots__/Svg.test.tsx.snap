// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Svg /> renders 1`] = `
<div
  className="css-k5inyr"
>
  <svg
    enableBackground="new 0 0 64 64"
    viewBox="0 0 64 64"
    xmlns="http://www.w3.org/2000/svg"
  />
</div>
`;

exports[`<Svg /> renders with children 1`] = `
<div
  className="css-k5inyr"
>
  <svg
    enableBackground="new 0 0 64 64"
    viewBox="0 0 64 64"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g />
  </svg>
</div>
`;
